content/docs/rfcs/14/README.md

```diff
@@ -1,120 +1,102 @@
   uint32 version = 3;
   sint64 timestamp = 10;
   bool ephemeral = 31;
 }
 ```
 
-## Payload encryption
-
-Payload encryption depends on the `version` field.
-
-### Version 0
+# Payload encryption
 
-This indicates that the payload SHOULD be either unencrypted or that encryption is done at a separate layer outside of Waku.
+The waku message payload MAY be encrypted. The message `version` attribute indicates the schema used to encrypt the payload data.
 
-### Version 1
+- **Version 0:**  The payload SHOULD be interpreted as unencrypted; additionally, it CAN indicate that the message payload has been encrypted at the application layer.
+- **Version 1:** The payload SHOULD be encrypted using Waku v1 payload encryption. [26/WAKU-PAYLOAD](/spec/26).
+This provides asymmetric and symmetric encryption. The key agreement is performed out of band. And provides an encrypted signature and padding for some form of unlinkability.
+- **Version 2:** The payload SHOULD be encrypted using [35/WAKU2-NOISE](/spec/35).
+Waku Noise protocol provides symmetric encryption and asymmetric key exchange.
 
-This indicates that payloads MUST be encrypted using [WAKU2-PAYLOAD](/spec/26).
+Any `version` value not included in the list is reserved for future use, and the payload SHOULD be interpreted as unencrypted.
 
-This provides for asymmetric and symmetric encryption.
-Key agreement is out of band.
-It also provides an encrypted signature and padding for some form of unlinkability.
+# Security Considerations
 
-### Version 2
+## Confidentiality, integrity, and authenticity
 
-This indicates that payloads MUST be encoded using [35/WAKU-NOISE](/spec/35).
+The level of confidentiality, integrity, and authenticity of the waku message payload is discretionary. Accordingly, the application layer shall utilize the encryption and signature schemes supported by Waku v2 to meet the application-specific privacy needs.
 
-This provides for symmetric encryption and
-asymmetric key-exchange protocols.
+## Reliability of the `timestamp` attribute
 
-# Differences from Whisper / Waku v1 envelopes
+The waku message `timestamp` attribute is set by the sender. Therefore, because message timestamps aren’t independently verified, this field is prone to exploitation and misuse. It should not solely be relied upon for operations such as message ordering.
 
-In Whisper and Waku v1, an envelope contains the following fields: `expiry, ttl, topic, data, nonce`.
+For example, a malicious actor can arbitrarily set the `timestamp` of a waku message to a high value so that it always shows up as the most recent message in a chat application. Applications using waku messages’ `timestamp` attribute are recommended to use additional methods for more robust message ordering. An example of how to deal with message ordering against adversarial message timestamps can be found in the Status protocol, see [6/PAYLOADS](https://specs.status.im/spec/6#clock-vs-timestamp-and-message-ordering).
```

Comment:
I think we need to settle on `Waku` as always being capitalised. @kaiserd, thoughts?